components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -595,5 +595,85 @@
   "captchaDismiss": {
     "message": "Dismiss",
     "description": ""
+  },
+  "braveTalkTurnOnRewardsToStartCall": {
+    "message": "To start a free call, turn on Brave Rewards",
+    "description": ""
+  },
+  "braveTalkBraveRewardsDescription": {
+    "message": "With Brave Rewards, you earn tokens for viewing private ads (your data is safe). Turn those tokens into cash, gift cards…or even tip the websites you love. No tracking. No slow downs.",
+    "description": ""
+  },
+  "braveTalkTurnOnRewards": {
+    "message": "Turn on Rewards",
+    "description": ""
+  },
+  "braveTalkOptInRewardsTerms": {
+    "message": "By clicking, you agree to the $tos_link_begin$Terms of Service$tos_link_end$ and $pp_link_begin$Privacy Policy$pp_link_end$. Disable any time in Settings.",
```

Comment:
Perhaps instead of "clicking" we can use a more inclusive term more specific to the action?

Reply:
Will be fixed in a separate issue: https://github.com/brave/brave-browser/issues/18445